Update ImagingSettings (#4510)
umbraco/UmbracoDocs: Reference/Configuration/ImagingSettings/index.md

## Cache

Contains configuration for browser and server caching. 

> When changing these cache headers, its recommended to clear your media cache. As meta data is stored in the cache and not updated when the configuration is changed.

Reviewer: ```suggestion
When changing these cache headers, it is recommended to clear your media cache. This is due to the data being stored in the cache and not updated when the configuration is changed.
```
